src/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.cs

```diff
@@ -41,7 +70,7 @@ public sealed class OptionalFieldAttribute : Attribute
     {
         public int VersionAdded
         {
-            get { return default(int); }
+            get { throw null; }
```

Comment:
Now that we are manually creating these cs files for reference assemblies I'm fine with using `throw null`. 

fyi @ericstj